addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3527,22 +3529,18 @@ msgstr ""
 
 #empty string with id 824
 
-#. Representation of PVR timer state "disabled"
-#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
-msgctxt "#825"
-msgid "Inactive"
-msgstr ""
+#empty string with id 825
 
-#. Notification text to announce that a timer was activated
+#. Notification text to announce that a timer was enabled
 #: xbmc/pvr/timers/PVRTimerInfoTag.cpp
 msgctxt "#826"
-msgid "Timer activated"
+msgid "Timer enabled"
 msgstr ""
 
-#. Notification text to announce that a timer was deactivated
+#. Notification text to announce that a timer was disabled
 #: xbmc/pvr/timers/PVRTimerInfoTag.cpp
 msgctxt "#827"
```

Comment:
As 19058 is "Timer enabled" and 19057 is unused (https://github.com/xbmc/xbmc/blob/master/addons/resource.language.en_gb/resources/strings.po#L8674), I think it is a good idea to make current 827 new 19057. (I introduced 827 with the series recording PR. It should not be used elsewhere)